RadarSDK/RadarLogger.m

```diff
@@ -42,4 +56,65 @@ - (void)logWithLevel:(RadarLogLevel)level type:(RadarLogType)type message:(NSStr
     });
 }
 
+- (void) logWithLevelLocal:(RadarLogLevel)level message:(NSString *)message {
+    [self logWithLevelLocal:level type:RadarLogTypeNone message:message];
+}
+
+- (void) logWithLevelLocal:(RadarLogLevel)level type:(RadarLogType)type message:(NSString *)message {
+    RadarLog *radarLog = [[RadarLog alloc] initWithLevel:level type:type message:message];
+
+    NSData *fileData = [self.fileHandler readFileAtPath:self.logFilePath];
+    NSMutableArray<RadarLog *> *existingLogs = [NSMutableArray array];
+    if (fileData) {
+        NSArray *jsonArray = [NSJSONSerialization JSONObjectWithData:fileData options:0 error:nil];
+        for (NSDictionary *jsonDict in jsonArray) {
+            RadarLog *existingLog = [[RadarLog alloc] initWithDictionary:jsonDict];
+            [existingLogs addObject:existingLog];
+        }
+    }
+    [existingLogs addObject:radarLog];
+
+    NSMutableArray *updatedLogsArray = [NSMutableArray array];
+    for (RadarLog *log in existingLogs) {
+        [updatedLogsArray addObject:[log dictionaryValue]];
+    }
+
+    NSData *updatedLogData = [NSJSONSerialization dataWithJSONObject:updatedLogsArray options:0 error:nil];
+    [self.fileHandler writeData:updatedLogData toFileAtPath:self.logFilePath];
+}
+
+- (void)flushLocalLogs {
```

Comment:
this is never called – do we want to have this, or instead have some mechanism to prune the logs in the write?

Reply:
I'm actually calling this on `initializeWithPublishableKey` to send flush all the logs on the file to the dashboard